templates/advantage/index.html

```diff
@@ -40,14 +40,34 @@ <h1>Ubuntu Advantage for&nbsp;Infrastructure</h1>
     <section class="p-strip is-shallow" style="overflow: visible;">
       <div class="row u-vertically-center">
         <div class="col-6 col-medium-3">
-          <h2>Your paid subscriptions</h2>
+          <h2>Your {% if subscriptions['total_subscriptions'] > 1 %}{{subscriptions['total_subscriptions']}} {%endif%}paid subscription{% if subscriptions['total_subscriptions'] == 1 %}{%else%}s{%endif%}</h2>
```

Comment:
Tiny comment but could do with spaces between the `%` and the operator. For example: `{% endif %}`

Reply:
Done! :+1: